etcd/etcd-live-http-to-https-migration.md

```diff
@@ -1,73 +1,73 @@
-# etcd Live HTTP to HTTPS migration
+# Enabling HTTPS in an Existing etcd Cluster
```

Comment:
This doc is in os because we use CoreOS related stuff here. Systemd drop-ins, `etcd2`, `update-ca-certificates`, etc.